hello_world/blockchain_app/hello_module/hello_asset.js

```diff
@@ -0,0 +1,66 @@
+const {
+    BaseAsset,
+    codec,
+    validator
+} = require('lisk-sdk');
+const { LiskValidationError } = require('@liskhq/lisk-validator');
+const {
+    helloCounterSchema,
+    CHAIN_STATE_HELLO_COUNTER
+} = require('./schemas');
+
+const HelloAssetID = 0;
+
+class HelloAsset extends BaseAsset {
+    name = 'helloAsset';
+    id = HelloAssetID;
+    schema = {
+        $id: '/hello/asset',
+        type: 'object',
+        required: ["helloString"],
+        properties: {
+            helloString: {
+                dataType: 'string',
+                fieldNumber: 1,
+            },
+        }
+    };
+
+    validate({asset}) {
+        if (!asset.helloString || typeof asset.helloString !== 'string' || asset.helloString.length > 64) {
+            throw new validator.LiskValidationError(
+                'Invalid "asset.hello" defined on transaction: A string value no longer than 64 characters is expected',
+                asset.helloString,
+            );
```

Comment:
It is not required to throw `LiskValidationError`, you can also use `Error`. As you are not using the `lisk-validator` to validate data, I suggest to keep it simple and throw simple error.